chsdi/views/files.py

```diff
@@ -165,8 +107,11 @@ def __init__(self, request):
                 self.file_id = id
             try:
                 key = self.bucket.get_key(self.file_id)
-            except:
-                raise exc.HTTPInternalServerError('Cannot access file with id=%s' % self.file_id)
+            except S3ResponseError as e:
+                raise exc.HTTPInternalServerError('Cannot access file with id=%s: %s' % (self.file_id, e))
+            except Exception as e:
+                raise exc.HTTPInternalServerError('Cannot access file with id=%s: %s' % (self.file_id, e))
```

Comment:
I thought to distinguish errors coming from S3 and otther erros.
S3ResponseError (meaning a request has been sent)
and
Exception (any other error)
text could be different.